src/libs/modulegroups.c

```diff
@@ -1830,9 +1825,11 @@ void init_presets(dt_lib_module_t *self)
   AM("tonemap");
   AM("vibrance");
   AM("basicadj");
-  // this modules are deprecated in 3.6 and should be removed 1 yer later
+  // these modules are deprecated in 3.6 and should be removed 1 year later
   AM("spots");
   AM("defringe");
+  // these modules are deprecated in 3.8 and should be removed 1 year later
+  AM("clipping");
```

Comment:
do we want to take the opportunity to "remove" 3.4 deprecated modules too ?
Or we can do it later...

Reply:
Later, in another PR as this will certainly get some discussions :)